dashboard/config/routes.rb

```diff
@@ -202,6 +202,9 @@ module OPS
   resources :followers, only: [:create]
   post '/followers/remove', to: 'followers#remove', as: 'remove_follower'
 
+  post '/followers/create_async', to: 'followers#create_async', defaults: {format: 'json'}
+  post '/followers/remove_async', to: 'followers#remove_async', defaults: {format: 'json'}
+
```

Comment:
Since these are API routes, please put them under [/api/v1](https://github.com/code-dot-org/code-dot-org/blob/e96aa5284dae1351a3f22405b4302058ec50221e/dashboard/config/routes.rb#L435-L436) in a new controller.